lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2707,6 +2708,299 @@ def calc_arrow(uvw, angle=15):
 
     quiver3D = quiver
 
+    def errorbar(self, x, y, z, zerr=None, yerr=None, xerr=None, fmt='',
+                 barsabove=False, errorevery=1, ecolor=None, elinewidth=None,
+                 capsize=None, capthick=None, xlolims=False, xuplims=False,
+                 ylolims=False, yuplims=False, zlolims=False, zuplims=False,
+                 **kwargs):
+        """
+        Draws error bars on an Axis3D instance.
+
+        Parameters
+        ----------
+        x : scalar
```

Comment:
Good point. I took and modified the docstring from the original 2d function in `axes/_axes.py`, and it was set there in this way. Seems to have been introduced here: https://github.com/matplotlib/matplotlib/commit/4d2c994a5c62214b6728447dc137534410d00f53